packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -210,19 +210,31 @@ export const comments: { readonly [key in TokenName]: string | null } = {
         'Font line height for the "Placeholder" base token',
     placeholderFallbackFontFamily:
         'Fallback font family for the "Placeholder" base token',
-    bodyEmphasizedFont: 'Font shorthand for the "Body_Emphasized" base token',
-    bodyEmphasizedFontColor: 'Font color for the "Body_Emphasized" base token',
+    bodyEmphasizedFont: 'Font shorthand for the "BodyEmphasized" base token',
+    bodyEmphasizedFontColor: 'Font color for the "BodyEmphasized" base token',
     bodyEmphasizedDisabledFontColor:
-        'Disabled font color for the "Body_Emphasized" base token',
-    bodyEmphasizedFontFamily:
-        'Font family for the "Body_Emphasized" base token',
-    bodyEmphasizedFontSize: 'Font size for the "Body_Emphasized" base token',
-    bodyEmphasizedFontWeight:
-        'Font weight for the "Body_Emphasized" base token',
+        'Disabled font color for the "BodyEmphasized" base token',
+    bodyEmphasizedFontFamily: 'Font family for the "BodyEmphasized" base token',
+    bodyEmphasizedFontSize: 'Font size for the "BodyEmphasized" base token',
+    bodyEmphasizedFontWeight: 'Font weight for the "BodyEmphasized" base token',
     bodyEmphasizedFontLineHeight:
-        'Font line height for the "Body_Emphasized" base token',
+        'Font line height for the "BodyEmphasized" base token',
     bodyEmphasizedFallbackFontFamily:
-        'Fallback font family for the "Body_Emphasized" base token',
+        'Fallback font family for the "BodyEmphasized" base token',
+    bodyEmphasized2Font: 'Font shorthand for the "BodyEmphasized_2" base token',
```

Comment:
We need to better capture it in the docs but the convention in use would name this bodyEmphasizedPlus1.

Where the "element" part of the definition captures the plus or minus part (is is docs): https://github.com/ni/nimble/blob/33f4878a8ae07e14e0241693836a088a00ed7c23/packages/nimble-components/CONTRIBUTING.md#token-naming

And the plus / minus convention was captured in this thread (should be added to docs): https://github.com/ni/nimble/pull/351#discussion_r812364438